index.html

```diff
@@ -0,0 +1,229 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>Mobile Version</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <section class="mobile">
+
+      <div class="head">
+      <h1 class="title">My Recent Works</h1>
+      <hr>
+    </div>>
+           
+      <div class="image"></div>
+
+
+      <section class="banner">    
+        <h2>Multi-Post Stories</h2>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
+        </p>
+        <ul class="bn-buttons">
```

Comment:
Please try to follow the given Figma design. Though your design is not bad, I think it is better to follow the given mobile design.

This button section on your mobile version doesn't look like the one in the design.  Please try adding the corresponding CSS property to stick to the design as much as possible.

Check the image below to have a better understanding.
![Capture](https://user-images.githubusercontent.com/75709904/144516879-cf104c51-ca9b-4e39-ba11-51a526294a5c.PNG)

Reply:
Improved.